README.md

```diff
@@ -1,24 +1,160 @@
-# Solutions Repo Template
-
-Give your project a relevant name and description in this README.
+# Apollo Router Node.js proxy migration strategy
 
 **The code in this repository is experimental and has been provided for reference purposes only. Community feedback is welcome but this project may not be supported in the same way that repositories in the official [Apollo GraphQL GitHub organization](https://github.com/apollographql) are. If you need help you can file an issue on this repository, [contact Apollo](https://www.apollographql.com/contact-sales) to talk to an expert, or create a ticket directly in Apollo Studio.**
 
-## Installation
+## Run the demo
+
+1. Clone this repository
+
+   ```sh
+   git clone https://github.com/apollosolutions/router-node-proxy
+   ```
+
+2. Build and run the services:
+
+   ```sh
+   docker compose up --build
+   ```
+
+   The `node` service will fail to start because it can't connect to Unleash, but that can be ignored at this step.
+
+3. Create the feature toggles:
+
+   1. The rollout feature toggle:
+      ```sh
+      curl --location --request POST 'http://localhost:4242/api/admin/projects/default/features' \
+        --header 'Authorization: *:*.unleash-insecure-api-token' \
+        --header 'Content-Type: application/json' \
+        --data-raw '{
+          "type": "release",
+          "name": "router-rollout",
+          "description": "",
+          "impressionData": false
+        }'
+      ```
+   2. The replay-on-error feature toggle:
+      ```sh
+      curl --location --request POST 'http://localhost:4242/api/admin/projects/default/features' \
+        --header 'Authorization: *:*.unleash-insecure-api-token' \
+        --header 'Content-Type: application/json' \
+        --data-raw '{
+          "type": "release",
+          "name": "router-replay-on-error",
+          "description": "",
+          "impressionData": false
+        }'
+      ```
```

Comment:
Any way to automate this by chance?

Reply:
i don't think so. all i could automate was configuring default API keys